SunshineAppExport.psm1

```diff
@@ -27,63 +30,95 @@ function SunshineExport {
-        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
-    
-        <UserControl.Resources>
-            <Style TargetType="TextBlock" BasedOn="{StaticResource BaseTextBlockStyle}" />
-        </UserControl.Resources>
-    
-        <StackPanel Margin="16,16,16,16">
-            <TextBlock Text="Enter your Sunshine apps.json path" 
-            Margin="0,0,0,8" 
-            VerticalAlignment="Center"/>
+<UserControl
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+    <UserControl.Resources>
+        <Style TargetType="TextBlock" BasedOn="{StaticResource BaseTextBlockStyle}" />
+    </UserControl.Resources>
+
+    <StackPanel Margin="16,16,16,16">
+        <TextBlock Text="Enter your Sunshine apps.json path" 
+        Margin="0,0,0,8" 
+        VerticalAlignment="Center"/>
+
+        <Grid>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="*"/>
+                <ColumnDefinition Width="Auto"/>
+            </Grid.ColumnDefinitions>
 
             <TextBox x:Name="SunshinePath"
-            Margin="0,0,0,8"
+            Grid.Column="0"
+            Margin="0,0,8,0"
             Width="320"
-            HorizontalAlignment="Left"
+            Height="36"
+            HorizontalAlignment="Stretch"
             VerticalAlignment="Top"/>
 
-            <TextBlock x:Name="FinishedMessage" 
-            Margin="0,0,0,24" 
-            VerticalAlignment="Center"
-            Visibility="Collapsed"/>
-
-            <Button x:Name="OKButton"
-            IsDefault="true"
+            <Button x:Name="BrowseButton"
+            Grid.Column="1"
             Width="72"
-            Height= "36"
-            HorizontalAlignment="Center"
-            Content="Start"/>
-        </StackPanel>
-    </UserControl>
+            Height="36"
+            HorizontalAlignment="Right"
+            Content="Browse"/>
+        </Grid>
+
+        <TextBlock x:Name="FinishedMessage" 
+        Margin="0,0,0,24" 
+        VerticalAlignment="Center"
+        Visibility="Collapsed"/>
+
+        <Button x:Name="OKButton"
+        IsDefault="true"
+        Height= "36"
+        Margin="0,5,0,0"
+        HorizontalAlignment="Center"
+        Content="Export Games"/>
+    </StackPanel>
+</UserControl>
 "@
+
     $reader = [System.Xml.XmlNodeReader]::new($xaml)
     $window.Content = [Windows.Markup.XamlReader]::Load($reader)
 
     $appsPath = "$Env:ProgramW6432\Sunshine\config\apps.json"
 
     $inputField = $window.Content.FindName("SunshinePath")
-    $inputField.Text = $appsPath    
+    $inputField.Text = $appsPath   
+    
+    # Attach a click event handler to the Browse button
+    $browseButton = $window.Content.FindName("BrowseButton")
+    $browseButton.Add_Click({
+            $openFileDialog = New-Object System.Windows.Forms.OpenFileDialog
+            $openFileDialog.InitialDirectory = $appsPath
```

Comment:
In my testing the "InitialDirectory" property doesn't work because the "$appsPath" isn't a directory, but a file. Instead we should strip "\apps.json" off the end.

Also, we shouldn't use the $appsPath variable, but instead the SunshinePath.Text, then it will always open to the directory the user typed in the text field, as $appsPath only gets updated when the user clicks "Export Games". I know we don't save off the user's last selected path, but at least if they go back to browse again, it'll open in the directory they last selected.

So maybe the line should look something like this:
```suggestion
            $openFileDialog.InitialDirectory = Split-Path -Path $window.Content.FindName("SunshinePath").Text -Parent
```